WordPress/Classes/Utility/Logging/WPCrashLoggingProvider.swift

```diff
@@ -24,3 +24,16 @@ class WPCrashLoggingProvider: CrashLoggingDataProvider {
         return TracksUser(userID: account.userID.stringValue, email: account.email, username: account.username)
     }
 }
+
+// MARK: - Static Property
+extension WPCrashLoggingProvider {
+
+    static var userHasOptedOut: Bool {
+        get {
+            return UserDefaults.standard.bool(forKey: UserOptedOutKey)
```

Comment:
By default `boolForKey` returns `false` if the key doesn't exist, which will leave crash logging enabled by default.

Just want to verify that we're good with that!

Reply:
Yes, deliberate decision.